pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "dcicutils"
-version = "6.7.0"
+version = "6.8.0"
```

Comment:
This is presumably an incompatible change, not strictly a bug fix. I'd bump the major version, which would mean older versions of the system if something isn't upgraded wouldn't try to fetch incompatible support if someone did poetry update. That's the reason for making major version bumps.